pkg/operator/ceph/object/rgw.go

```diff
@@ -151,6 +157,8 @@ func (c *clusterConfig) startRGWPods(realmName, zoneGroupName, zoneName string)
 		if err != nil {
 			// Getting EPERM typically happens when the flag may not be modified at runtime
 			// This is fine to ignore
+			// No it is not: As errors in the mapKeystoneSecretToConfig function are returned
+			// directly and the situation may occur, that only part of the configuration is set
```

Comment:
I'm not sure what to do about this, but there seems to be a legitimate reason to allow EPERM for the current config values.

One option we have is to use a different instance of `c.setFlagsMonConfigStore(keystoneConfigs)` if the keystone configs need to handle this error condition differently. How does that sound?

Or am I missing something?

Reply:
I talked to the colleague who originally wrote this comment two years ago when he started out with this feature and he doesn't remember his reasoning from back then. I tried to reconstruct what he meant but I don't come to the same conclusion. When we took over the PR we left the comment in because we thought that maybe you from upstream would know what to do with it. It's quite possible though that the comment was from a first iteration and the problem was fixed afterwards.

As it stands now, I suggest we remove the comment and leave this check as it is. What do you think?

Reply:
I agree. It will be safest for existing users and configurations to keep this as it is.

If you find that there are issues coming up related to this in some Swift installs, we can revisit in the future. I hope we don't have to do that. Ceph doesn't really return useful exit codes that help distinguish different failure conditions, so this could get confusing.